cmd.sh

```diff
@@ -24,28 +24,65 @@ set -o nounset
 # POCKET_CORE_SERVICE_IP = IP of the Pocket Core service node (required)
 # POCKET_CORE_SERVICE_PORT = Port of the Pocket Core service node (required)
 
+
+
+
+
+
 # Start pocket-core
 if [ $POCKET_CORE_NODE_TYPE = "dispatch" ]; then
-	echo 'Starting pocket-core dispatch'
+	if [ ${POCKET_CORE_INTEGRATION_TESTS:-false}  == false ]; then
+		echo 'Starting pocket-core dispatch'
+	    
+		exec pocket-core --dispatch \
+		  --datadirectory ${POCKET_PATH_DATADIR:-datadir} \
+		  --dbend ${POCKET_CORE_AWS_DYNAMODB_ENDPOINT:-dynamodb.us-east-1.amazonaws.com} \
+		  --dbtable ${POCKET_CORE_AWS_DYNAMODB_TABLE:-dispatch-peers-staging} \
+		  --dbregion ${POCKET_CORE_AWS_DYNAMODB_REGION:-us-east-1} \
+		  --disip ${POCKET_CORE_DISPATCH_IP:-127.0.0.1} \
+		  --disrport ${POCKET_CORE_DISPATCH_PORT:-8081}
+
+	else
+		echo 'Initializing integration testing'
+		echo 'Starting pocket-core dispatch'
+		
+		nohup pocket-core --dispatch \
+		  --datadirectory ${POCKET_PATH_DATADIR:-datadir} \
+		  --dbend ${POCKET_CORE_AWS_DYNAMODB_ENDPOINT:-dynamodb.us-east-1.amazonaws.com} \
+		  --dbtable ${POCKET_CORE_AWS_DYNAMODB_TABLE:-dispatch-peers-staging} \
+		  --dbregion ${POCKET_CORE_AWS_DYNAMODB_REGION:-us-east-1} \
+		  --disip ${POCKET_CORE_DISPATCH_IP:-127.0.0.1} \
+		  --disrport ${POCKET_CORE_DISPATCH_PORT:-8081} &
+
+		sleep 10 
+		exec go test ./tests/integration/... --dispatch --dispatchtesturl dispatch.staging.pokt.network --servicetesturl service.staging.pokt.network
```

Comment:
Can we pass the URL's as environment variables?

Reply:
Oh, sorry. It's Done already